tests/console/check_os_release.pm

```diff
@@ -0,0 +1,79 @@
+# SUSE's openQA tests
+#
+# Copyright © 2017-2019 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: check contents of /etc/os-release per the current settings
+# Maintainer: Alvaro Carvajal <acarvajal@suse.com>
+
+use base "consoletest";
+use strict;
+use warnings;
+use testapi;
+use version_utils qw(is_sle is_leap is_tumbleweed is_sles4sap is_rt is_hpc is_sled);
+
+sub run {
+    my ($self) = @_;
+
+    select_console 'root-console';
```

Comment:
Yes, will do that.

Reply:
Didn't like how results are presented: http://mango.suse.de/tests/926#step/check_os_release/1

Will switch back to `select_console 'root-console'`.

Reply:
I still would `$self->select_serial_terminal` as it would be faster (and less prone to missing keys on certain architectures) but I see no reason to insist for now